backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -0,0 +1,25 @@
+[id="backing-up-applications"]
+= Backing up applications
+include::modules/common-attributes.adoc[]
+:context: backing-up-applications
+
+toc::[]
+
+You back up applications by xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[creating a `Backup` custom resource (CR)]. The `Backup` CR creates backup files for Kubernetes resources and internal images and snapshots for persistent volumes (PVs), if the prerequisites are met.
+
+:FeatureName: The cloud storage API for backups
+include::modules/technology-preview.adoc[]
+
+You can back up PVs by using the xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Snapshot Interface (CSI)] if the cloud provider supports CSI and does not have a default Velero plugin, for example, OpenShift Container Storage 4.
```

Comment:
I believe that it is not just CSI, it is CSI Snapshots which is an optional capability that CSI drivers may or may not support.

Reply:
I also wonder if there is a CSI section in the docs that we can link to that would be helpful to users as well?

Reply:
> I also wonder if there is a CSI section in the docs that we can link to that would be helpful to users as well?

CSI section in which docs? OCS? Outside Red Hat docs?

Reply:
> I believe that it is not just CSI, it is CSI Snapshots which is an optional capability that CSI drivers may or may not support.

Good point. I didn't realize that the snapshots were optional for CSI drivers. Will fix.

Reply:
https://docs.openshift.com/container-platform/4.9/storage/container_storage_interface/persistent-storage-csi-snapshots.html is what I was thinking??